.github/workflows/validate_tests.yml

```
@@ -0,0 +1,20 @@
+name: Validate test data
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: '**'
+
+jobs:
+  run_all:
+    name: Validate tests for specific schema version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Validate tests for schema v0.0.1
+        run: |
+            cd test
+            python3 validate.py --schema-dir ./schemas/v0-0-1/
```

With something like this, we don't need to define a custom validator script:

```suggestion
      - run: npm install --global ajv-cli
      - run: >
          ajv validate --spec=draft2020
          -s schemas/v0-0-1/tests.schema.json
          -d 'tests/*.json'
          -d 'tests/functions/*.json'
        working-directory: ./test
```

---

I like the idea of reusing a tool and not having to write a script. But having to enumerate each nested directory is annoying. Also, do we want a command to automatically figure out our latest schema version? The `ajv` tool only takes a hard coded schema version.

---

Oh, the `**` segment in the glob pattern takes care of nested directories, so that's not an issue. The only issue is whether we want to have this CI job automatically take the latest schema version, or if we want to manually maintain the CI job to reflect the latest schema every time we change it.